docs/user_guide/Upgrade_SIMP/General_Upgrade_Instructions.rst

```diff
@@ -108,9 +108,16 @@ steps as ``root``:
    ** **This ends the steps that are only for 6.4 or later.**  The next steps apply
    to all systems.
 
+#. Update the generated types for the environment
+
+   .. code-block:: sh
+
+     puppet generate types  --environment test
```

Comment:
```suggestion
     puppet generate types  --environment production
```

Reply:
Did you need to set your `umask` for this? I don't quite remember.

If we have them fire off `simp_generate_types` on all spaces it should "do the right thing"